website/src/blogs/logical-volume-management-lvm-on-kubernetes-with-openebs-localpv.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Logical Volume Management (LVM) on Kubernetes with OpenEBS LocalPV
+author: Pawan Prakash Sharma
+author_info: It's been an amazing experience in Software Engineering because of my love for coding. In my free time, I read books, play table tennis and watch tv series
+date: 17-06-2021
+tags: Kubernetes, OpenEBS, LocalPV, LVM
+excerpt: In large production environments, various physical disks are typically pooled to handle user and application storage needs. Logical Volume Management (LVM) is a storage virtualization technology that is a part of Linux which is widely used as a way to help administrators and developers abstract the physical configuration of storage devices for flexible and simpler provisioning. 
+not_has_feature_image: false
```

Comment:
since we have feature images, we don't need to use defaultImage

![Screenshot 2021-09-17 at 2 24 24 PM](https://user-images.githubusercontent.com/6524419/133754731-f1de15e4-5bb0-4075-a872-da263472799d.png)

Reply:
Yes, that is why `not_has_feature_image` is set to false. So it means it has a feature image